src/modules/userMethods.ts

```diff
@@ -339,6 +340,38 @@ export const userMethods = {
         });
     },
 
+    editUserFields: async (data: EditUserFields, user: User) => {
```

Comment:
looks like its an extension of edit method with backwards compatibility, we can just reuse edit method with new fields in schema